public/app/features/logs/components/panel/LogList.tsx

```diff
@@ -141,6 +171,18 @@ export const LogList = ({
   );
 };
 
+function getStyles(dimensions: LogFieldDimension[], { showTime }: { showTime: boolean }) {
+  const columns = showTime ? dimensions : dimensions.filter((_, index) => index > 0);
+  return {
+    logList: css({
+      '& .unwrapped-log-line': {
```

Comment:
Do we need the hardcoded class name?

Reply:
Yeah, I'm using it to target the fields wrapper when the line is unwrapped, setting the columns from the parent styles. I did it this way to avoid defining this for every single log line in display.